proto/viam/app/packages/v1/packages.proto

```diff
@@ -56,7 +57,6 @@ message PackageInfo {
   PackageType type = 4;
   repeated FileInfo files = 5;
   google.protobuf.Struct metadata = 6;
-  google.protobuf.Timestamp created_on = 7;
```

Comment:
I would remove this in a separate PR since it's a breaking change.

Reply:
Yup, added a ticket as a reminder